RapidReact/src/main/java/frc/robot/subsystem/BitBucketsSubsystem.java

```diff
@@ -0,0 +1,57 @@
+package frc.robot.subsystem;
+
+import com.ctre.phoenix.motorcontrol.can.BaseTalon;
+import edu.wpi.first.util.sendable.Sendable;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class BitBucketsSubsystem extends SubsystemBase
+{
+    private final List<BaseTalon> motors;
+
+    protected BitBucketsSubsystem()
+    {
+        this.setName(this.getClass().getSimpleName());
+
+        this.motors = new ArrayList<>();
+        this.addMotorsToList();
+    }
+
+    //When the subsystem is initialized
+    public abstract void init();
+
+    //Periodically called
+    public abstract void periodic();
+
+    //When the subsystem is turned off
+    public abstract void disable();
+
+    //Add motors to list (use this#addMotors() to make it simpler)
+    public abstract void addMotorsToList();
+
+    //Helper method that can be used in an implementation of this#addMotorsToList()
+    protected void addMotors(BaseTalon... motors)
+    {
+        this.motors.addAll(List.of(motors));
+    }
```

Comment:
I'm not sure we'll need these because I'm not sure we want the CANChecker again (given the network issues we had), but we can keep it for now.